### DISCUSSION TITLE
S4Studio Icon

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1352] [Icon request]
Would be great if someone could make this icon 🙏
https://64.media.tumblr.com/avatar_21f190ff90d4_128.pnj

**Comment by Arstidir** [comment]
I made it for you :)
https://macosicons.com/#/u/Rembihnutur

**Comment by Murplyy** [comment]
I happened to just make one for myself that is slightly more shaded at the bottom rather than a plain white background if you're interested- https://macosicons.com/#/u/baer
